osquery/process/windows/process_ops.cpp

```diff
@@ -76,9 +76,11 @@ uint32_t getGidFromSid(PSID sid) {
   if (accountFound == 0) {
     return static_cast<uint32_t>(-1);
   }
-  // USER_INFO_3 struct contains the RID (uid) of our user
-  DWORD userInfoLevel = 3;
```

Comment:
Interesting that this was set to 3 given the documentation

> Return detailed information and additional attributes about the user account. **This level is valid only on servers.** The bufptr parameter points to a USER_INFO_3 structure. Note that it is recommended that you use USER_INFO_4 instead. 

https://docs.microsoft.com/en-us/windows/win32/api/lmaccess/nf-lmaccess-netusergetinfo

Reply:
Yep, 4 will work fine (see other comment). I searched the codebase, and we don't seem to use the level 3 struct anywhere else either.